lib/LaTeXML/Post/UnicodeMath.pm

```diff
@@ -0,0 +1,435 @@
+  my $pos    = (ref $op && $op->getAttribute('scriptpos')) || 'post';
+  my $pre    = $pos =~ /^pre/;
+  my $mid    = $pos =~ /^mid/;
+  my $mapped = 0;
+  my $ubase  = ($base ? unimath_nested($base, $PREC_SCRIPTOP) : '');
+  my ($uscript, $prec) = unimath_internal($script);
+  # Possibly convert $uscript to superscript codepoints ??
+  if ($mid) { }    # Could try for combining accents?
+  else {
+    my $u_uscript = unicode_convert($uscript, 'superscript');
+    if (defined $u_uscript) {
+      $uscript = $u_uscript; $mapped = 1; } }
+  $uscript = '{' . $uscript . '}' if !$mapped && $prec < $PREC_SCRIPTOP;    # Wrap if needed
+  return (($pre ? ($mapped ? $uscript : '{^' . $uscript . '}') . $ubase
+      : $ubase . ($mapped ? '' : ($mid ? "\x{2534}" : '^')) . $uscript), $PREC_SCRIPTOP); }
+
+# The combining chars assciated with the accent chars over/under
+# Perhaps these tables should be moved to LaTeXML::Util::Unicode? (later...)
+our %overaccents = ('^' => "\x{0302}",    # \hat,
+  UTF(0x5E)  => "\x{0302}",               # \widehat
+  "\x{02C7}" => "\x{030C}",               # \check
+  '~'        => "\x{303}",
+  UTF(0x7E)  => "\x{0303}",               # \tilde, \widetilde
+  UTF(0x84)  => "\x{0301}",               # \acute
+  UTF(0x60)  => "\x{0300}",               # \grave
+  "\x{02D9}" => "\x{0307}",               # \dot
+  UTF(0xAB)  => "\x{0308}",               # \ddot
+  UTF(0xAF)  => "\x{0304}",               # \bar, \overline
+  "\x{2192}" => "\x{20D7}",               # \vec
+  "\x{02D8}" => "\x{0306}",               # \breve
+  "o"        => "\x{030A}",               # \r
+  "\x{02DD}" => "\x{030B}",               # \H
+);
+our %underaccents = (
+  UTF(0xB8) => "\x{0327}",                # \c
+  '.'       => "\x{0323}",                # dot below
+  UTF(0xAF) => "\x{0331}",                # macron below
+  "="       => "\x{0361}",                # \t
+  ","       => "\x{0361}",                # lfhook
+);
+# \overbrace, \underbrace, \overleftarrow ??
+sub unimath_overaccent {
+  my ($op, $base) = @_;
+  my $acc  = $op->textContent;
+  my $cacc = $acc && $overaccents{$acc};
+  my ($ubase, $uprec) = unimath_internal($base);
+  $ubase = '(' . $ubase . ')' if length($ubase) > 1;
+  return ($ubase . ($cacc ? $cacc : "\x{252C}" . $acc), $PREC_SCRIPTOP); }
+
+sub unimath_underaccent {
+  my ($op, $base) = @_;
+  my $acc  = $op->textContent;
+  my $cacc = $acc && $underaccents{$acc};
+  my ($ubase, $uprec) = unimath_internal($base);
+  $ubase = '(' . $ubase . ')' if length($ubase) > 1;    # maybe NBSP, too???
+  return ($ubase . ($cacc ? $cacc : "\x{252C}" . $acc), $PREC_SCRIPTOP); }
+
+# Handle text contents.
+# We probably should pass this back to the same code used in CrossRef.
+# But also, there's the question of what to do with nested math?
+# Is that even allowable in UnicodeMath?
+sub unimath_text {
+  my (@nodes) = @_;
+  return ('"' . join('', map { (ref $_ ? $_->textContent : $_); } @nodes) . '"', $PREC_SYMBOL); }
+
```

Comment:
Indeed, both `\binom` and `\text` need to get recognized... by something like `--preload=amsmath.sty`, perhaps? :>

Reply:
> Indeed, both `\binom` and `\text` need to get recognized... by something like `--preload=amsmath.sty`, perhaps? :>

Ack! This is even more frustrating given that I was checking the XMath with `latxmlc --preload=amsmath.sty` in my other tab. For some reason I must have gotten distracted.

Well, if those two look good with amsmath.sty actually loaded, feel free to merge! We should get the `--unicodemath` math post-processor flag for latexmlc and latexmlpost eventually too, maybe then I'll manage to get my options right.

Reply:
Well, since you insist :>  Since it fits the same pattern as `--mathtex` it's easy to add; But still check that I've got the `latexmlc` stuff right.

(and one of my use cases wants to convert hundreds of little math fragments which cries out for some kind of daemon approach)